sh/install.sh

```diff
@@ -43,6 +44,39 @@ function exit_with_failure() {
     exit_with_message "FAILURE: $1" 1
 }
 
+function install_python_rhel() {
+        file_name=$1
+        # Get the redhat version
+        major=$(cat $file_name | tr -dc '0-9.'|cut -d \. -f1)
+        if [ $major -le 7 ]
+        then
+            yum install -y python >> ${INSTALL_LOG} 2>&1
+            if [ $? -eq 0 ]
+            then
+                    echo "Installed python"
+            else
+                    echo "Failed to install python"
+            fi
+        else
+            PYTHON_PATH=$(which python2)
```

Comment:
python 2 will not be supported after the end of this year. Could we look into using python 3 instead?

Reply:
As of now wavefront CLI only supports python 2.
Is it fine to add the python 3 support for Wavefront CLI as a different task?

Reply:
@yogeshprasad Have you tried using python3? What is the failure?

The python2 EOL is coming very soon. We should try to understand what the issue with python3 is and fix it. Specifically installing python2 is not a good idea.